### PR TITLE
add popular plugins post

### PR DESCRIPTION
**Pull request by andrea-p**

### CODE REVIEW

File: posts/2018-01-22-most-popular-framework-plugins.md [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+A cool walkthrough that uses it:
+- [Writing a Serverless app with full local development experience](https://serverless.com/blog/event-driven-serverless-app-local-dev-exp/)
+
+---
+
+<img src="https://avatars3.githubusercontent.com/u/30321405?s=200&v=4" align="left" height="100px">
+
+**3. [Serverless Webpack](https://github.com/serverless-heaven/serverless-webpack)**, by [Serverless Heaven](https://github.com/serverless-heaven)
+
+If you want to use the latest Javascript version with Babel, this plugin is for you! Use custom resource loaders, optimize your packaged functions individually, and more.
```
Comment (HyperBrain) [inline posts/2018-01-22-most-popular-framework-plugins.md]: Hi @andrea-p , I would mention the ability to use it for arbitrary transpiled languages too. Noting that only the latest JS can be used seem a bit "flat" :-)
So I would mention the support of them explicitly. Samples for them are: TypeScript or ELM (!) - cool demo at https://github.com/ktonon/elm-serverless-demo and many more. In fact anything that is supported by Babel.

**Comment by HyperBrain** [comment]
@andrea-p I'm not sure if this blog post would be the right place, but we also have some editor integrations (which improve UX when using Serverless) available recently, made by the @serverless/framework-maintainers . They could be mentioned in a separate section - or is this completely off-topic?

The extensions are:
@horike37 [Atom Serverless Dashboard](https://github.com/horike37/serverless-dashboard-for-atom)
@RafalWilinski [VSCode Serverless snippets](https://github.com/RafalWilinski/serverless-vscode-snippets) [VSCode extension](https://marketplace.visualstudio.com/items?itemName=rafwilinski.serverless-vscode-snippets)
@HyperBrain [VSCode Serverless integration](https://github.com/HyperBrain/serverless-vscode) => [VSCode extension](https://marketplace.visualstudio.com/items?itemName=frankschmid.serverless-vscode)

**Comment by horike37** [comment]
Nice post @andrea-p :+1: 
I think it would be good to introduce `serverless plugin install` command instead of `npm install` when installing plugins. You can automatically add the plugin setting into serverless.yml file.

File: posts/2018-01-22-most-popular-framework-plugins.md (hunk 1 above)
Comment (andrea-p) [reply posts/2018-01-22-most-popular-framework-plugins.md]: Thanks @HyperBrain! I'll make these changes.

**Comment by andrea-p** [comment]
@HyperBrain re: editor integrations, we actually might do a different post on this in a few weeks :)

@horike37 good call, I'll update that!

**Comment by rupakg** [comment]
@andrea-p Please update every `serverless install plugin xxxxxx` command with `serverless install plugin -n xxxxxx`. Otherwise it will not work.

**Comment by andrea-p** [comment]
schedule(01-22-2018 6:00am)

**Comment by DavidWells** [comment]
✅ Post scheduled for **01-22-2018 at 6:00amundefined (America/Los_Angeles)**

**To unscheduled the post:**

1. **[Delete the previous comment ⬆](https://github.com/serverless/blog/pull/489#issuecomment-359075436)** with schedule(time) information by clicking the ❌
  ```
  ┌──────────────────────────────────────────────┐
  │                                           X  │
  ├───────────────────────────────────────────▲──┤
  │                                           │  │
  │  schedule(mm/dd/yyyy h:mm A)   ┌────────┐ │  │
  │                                │ Delete ├─┘  │
  │                                └────────┘    │
  └──────────────────────────────────────────────┘
  ```
2. (Optionally) Delete this comment to keep things tidy

> Powered by the ⚡️ [Serverless post scheduler](https://github.com/serverless/post-scheduler) | Made with ❤️  by [Serverless](https://serverless.com/)

**Comment by DavidWells** [comment]
🎉 **The post has been published**

> Powered by the ⚡️ [Serverless post scheduler](https://github.com/serverless/post-scheduler) | Made with ❤️  by [Serverless](https://serverless.com/)

**Pull request merged**